kivy/tools/report.py

```diff
@@ -60,8 +60,8 @@ def send_report(dict_report):
                 "content": "\n".join(dict_report['Configuration']),
                 "type": 'text'
             },
-            "Input Availablity.txt": {
-                "content": "\n".join(dict_report['InputAvailablity']),
+            "Input Availability.txt": {
+                "content": "\n".join(dict_report['InputAvailability']),
```

Comment:
I'm just bringing these to you attention. I can revert if necessary.

Reply:
i think it's fine to fix this one as well, it's just a tool, not part of the kivy api, not worth keeping an error around for the sake of backward compatibility :)